### PR TITLE
Started grammar and puncuation editing

### PR DESCRIPTION
**Pull request by KelvinShadewing**
Started going through files and editing the contents for grammar, punctuation, and spelling. More fixes to come later; it's bed time for me now.

### CODE REVIEW

**Comment by fluteds** [comment]
Is this done with American English rules and spelling? I'm happy to help out if you need some extra eyes!

File: src/aelplermagronen.md [hunk 1]
```diff
@@ -19,19 +19,20 @@ Cooking time: ~30 minutes
...
 8. Shred your cheese.
-9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough it is most likely due to using the wrong cheese.
+9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough, it is most likely due to using the wrong cheese.
 10. Season to taste. (Needs quite a bit of salt). Nutmeg also works well here.
 11. Serve with apple sauce. Should be eaten together, not as a dessert.
 
 ## Contributors
 
 - **Alexander Bocken** -- [contact](mailto:alexander@bocken.org)
+- Kelvin Shadewing - https://odysee.com/@KelvinShadewing:d
```
Comment (Holsterbau) [inline src/aelplermagronen.md:36]: Should be `[Channel](https://odysee.com/@KelvinShadewing:d)`

**Review by hallzy — APPROVED** [review]

**Comment by KelvinShadewing** [comment]
Removed contribution links from minor edits. Figured it'd get obnoxious crediting myself just for a few missing periods and commas everywhere. Also, I fixed the issues the system mentioned, but it's still saying my tofu stir fry recipe is failing checks, just won't say why now.

**Comment by KelvinShadewing** [comment]
Does anyone know why my recipe keeps getting rejected? I've compared it to several others and can't find the difference that's making the compliance check fail.

**Comment by Socerest2** [comment]
> Does anyone know why my recipe keeps getting rejected? I've compared it to several others and can't find the difference that's making the compliance check fail.

Yeah It looks like to much spacing, I'll fix em up.

File: src/arroz-chaufa.md [hunk 2]
```diff
@@ -23,13 +23,13 @@ Peruvian-chinese dish, Easy to cook just add and mix everything.
```
Comment (Socerest2) [inline src/arroz-chaufa.md:22]:
```suggestion
```
Fix Spacing / Why Check Is Failing

**Comment by KelvinShadewing** [comment]
Finally got it! Heck, this thing sure is strict about spacing. Also sucks I couldn't have "Simple Cooking Procedures" instead of "Directions". Oh well. XD

File: src/assam-tea.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ This is a simple Assam tea recipe.
 1. Add 150 ml of water to a small pan and bring it to boil.
 2. Then add 2 teaspoons of tea into it and reduce the heat to low.
 3. Strain the tea using a tea filter into a cup.
-4. Add sugar and milk to taste
+4. Add sugar and milk to taste.
```
Comment (Socerest2) [inline src/assam-tea.md]:
```suggestion

```
Fixing Spacing / Why Check Is Failing

**Review by Socerest2 — CHANGES_REQUESTED** [review]
Spacing fix

**Review by Socerest2 — APPROVED** [review]
Look good now!

File: src/aelplermagronen.md [hunk 4]
```diff
@@ -19,19 +19,20 @@ Cooking time: ~30 minutes
...
+1. Fry bacon cubes in pot (this pot will be used for everything so choose an appropriately large one).
 2. Cut onions into half-rings and let them sweat in the same pot. Add some butter if your bacon was not fatty enough.
-3. Peel potatoes and cut them into ~1 cm/half inch cubes
+3. Peel potatoes and cut them into ~1 cm/half inch cubes.
 4. When the onions have become sufficiently cooked, add potatoes.
 5. Top everything with milk and let the potatoes cook for about 10 minutes.
 7. Add macaroni and the remaining milk to cover everything. Most of the milk will be absorbed by the macaroni.
 8. Shred your cheese.
-9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough it is most likely due to using the wrong cheese.
+9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough, it is most likely due to using the wrong cheese.
```
Comment (fluteds) [inline src/aelplermagronen.md:29]: I would suggest changing the `the macaroni are done` to `the macaroni is done` as it makes more grammatical sense

Comment (KelvinShadewing) [reply src/aelplermagronen.md:29]: IIRC, macaroni is also plural, but it does sound better using 'is' instead of 'are'. Thanks.

**Pull request merged**